<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interNewVente</class>
 <widget class="QMainWindow" name="interNewVente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Microsoft Yi Baiti&quot;;</string>
    </property>
    <property name="text">
     <string>Nouvelle vente</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1011</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>541</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Infomation sur le client</string>
    </property>
    <widget class="QPushButton" name="btnReload">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Rafraichir</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/Usual/Refresh/refresh.png</normaloff>img/Usual/Refresh/refresh.png</iconset>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbPatient">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>351</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNew">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Nouveau</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/Usual/New or add/add_file.png</normaloff>img/Usual/New or add/add_file.png</iconset>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date d'achat</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupVerre">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>541</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Informations sur les verres</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>521</width>
       <height>221</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Oeil gauche</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Verre (VL)</string>
       </property>
      </widget>
      <widget class="QComboBox" name="OgVergVl">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Traitements</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="OgTrait">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>411</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Verre (VP)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_49">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Qte</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="OgQteVl">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_50">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>10</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="OgPuVl">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>10</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="OgPuVp">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>50</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_51">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Qte</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_52">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>50</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QComboBox" name="OgVergVp">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="OgQteVp">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>50</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Oeil droit</string>
      </attribute>
      <widget class="QTextEdit" name="OdTrait">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>411</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_59">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>10</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="OdPuVp">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>50</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="OdQteVp">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>50</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QComboBox" name="OdVergVp">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_53">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Verre (VP)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_57">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Qte</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="OdQteVl">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_58">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Traitements</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_56">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Qte</string>
       </property>
      </widget>
      <widget class="QComboBox" name="OdVergVl">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="OdPuVl">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>10</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_55">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Verre (VL)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_54">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>50</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupOther">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>461</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Autres</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Libelle</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtOther">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>341</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Prix</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="puOther">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombres</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>90</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>F CFA</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="nbrOther">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnAnnul">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnValider">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Terminer</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupMonture">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Monture</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <widget class="QComboBox" name="cmbMont">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>341</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Caracteristique</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>F CFA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombres</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Prix</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="puMon">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="nbrMont">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupNettoyant">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>461</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Nettoyant</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Caracteristques</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbNett">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>341</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Prix</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombres</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="puNet">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>F CFA</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="nbrNet">
     <property name="geometry">
      <rect>
       <x>111</x>
       <y>60</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
